themes/default/content/blog/introducing-azure-native-v2/index.md

```diff
@@ -0,0 +1,111 @@
+
+## Reduced SDK Size for Faster Deployments
+
+We've listened to your feedback and have made significant optimizations to the Azure Native Provider SDKs. With careful fine-tuning and restructuring, we have achieved a remarkable decrease in SDK size of up to 60% <!-- TODO: Get latest size updates on alpha.3 once released -->. This improvement translates into faster deployment times, reduced resource consumption, and an overall smoother experience when working with the provider.
+
+So, how did we accomplish this? For starters, we've removed all deprecated ARM API resources <!-- TODO: Link to list of deprecated resources --> up to the 2.0 release date. The bigger impact was made by identifying and removing all forward-compatible resource versions. That is to say we've exluded any redundant versions and kept the latest version until breaking changes. Where that version is not the current latest and new default, the explicit versions are still available.
+
+![version_squeeze_diagram](./azurev2-diagrams.png)
+
+## Refreshed Default Resource Versions
+
+In the latest release, we have refreshed the default resource versions to align with the latest ARM API versions from Microsoft. With our continued committment to same-day access to Azure API surface, you always have access to the most up-to-date features and capabilities offered by Azure resources. By leveraging the latest resource versions, you can take advantage of the latest and most desired features directly from Azure while building your infrastructure with Pulumi.
+
+If you would like to upgrade to 2.0, but continue using the same ARM API version that was used in your 1.0, you can do that by specifying the compatible explicit version available in the 2.0 provider. Shown below are examples of how that change might look.
+
+{{< chooser language "typescript,python,csharp,go,yaml" >}}
+
+{{% choosable language typescript %}}
+
+```typescript
+- import { EventSubscription } from "@pulumi/azure-native/eventgrid";
++ import { EventSubscription } from "@pulumi/azure-native/eventgrid/v20200601";
+```
+
+{{% /choosable %}}
+{{% choosable language python %}}
+
+```python
+- from pulumi_azure_native import eventgrid
++ from pulumi_azure_native.eventgrid import v20200601 as eventgrid20200601
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+```csharp
+- using EventGrid = Pulumi.AzureNative.EventGrid;
++ using EventGrid20200601 = Pulumi.AzureNative.EventGrid.V20200601;
+```
+
+{{% /choosable %}}
+{{% choosable language go %}}
+
+```go
+- import eventgrid "github.com/pulumi/pulumi-azure-native-sdk/eventgrid/v2"
++ import eventgrid20200601 "github.com/pulumi/pulumi-azure-native-sdk/eventgrid/v2/v20200601”
+```
+
+{{% /choosable %}}
+{{% choosable language yaml %}}
+
+```yaml
+- type: azure-native:eventgrid:Topic
++ type: azure-native:eventgrid/v20200601:Topic
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
```

Comment:
Should we call out the path conflict resolution that @thomas11 did? Or maybe the effect of it - e.g. that the default version of MySQL/Postgres is pointing to Flexible Servers.

Reply:
This will be included in the migration guide. I'm okay leaving it out here.